### ISSUE TITLE
How to edit prelabeled data?

### ISSUE THREAD

**Issue by Skullface9512** [diegoaldarondo/Label3D#27]
Hi, I'm studying on a dataset production work , and i think it is very time consuming that all the labels of joints are annotated by human，so i want to annotate a part of the data by some keypoints detection model at first, then correct them by human, but i don't know how to import prelabeled 2D joints data into the project, can you give me some advice, thank you very much.

**Comment by diegoaldarondo** [comment]
Hi! 

With 3D key point data it's pretty easy to load existing key point estimates from some model. Check out the loadFrom3D class function, which accepts a matrix of 3d key points and incorporates then into a running label3d object. 

For existing 2d key point data it will be a little more difficult. I'd recommend manually overwriting the camPoints class attribute with the relevant frame/camera/key points. You also might need to update the relevant frames and keypoints in the status class attribute to 1 to denote that those fromes have been initialized. If you have multiview 2d data I would recommend triangulating first then loading with 3d triangulated data. 

@caxon Any thoughts on this and on adding a 2d loading function in the future?

**Comment by Skullface9512** [comment]
Thank you for giving me some useful advice. The followings are some attempts I have made.
For 3D key point data, I tried to load 3d key points to the label3d object, and it worked.
For 2D key point data, I attempted to give some random data to the variable 'camPoints' and 'handLabeled2D', and set 'isLabeled',  'isInitialized', the 'status' of corresponding joint,camera and frame to 1. When open the label3d GUI, I found that there's no point in the figure. When I clicked 't', all the elements in 'obj.status' matrix were set to 0. Is there something ignored in these steps?
